tests/integration/test_github.py

```diff
@@ -6,6 +6,16 @@
 from ogr.abstract import PRStatus
 from ogr.services.github import GithubService
 
+def skip_tests(env_vars_list):
+    requirements_met = True
+    for item in env_vars_list:
+        if os.environ.get(item) is None:
+            requirements_met = False
+    return pytest.mark.skipif(not requirements_met,
+                              reason="you have to have set env vars: {}".format(env_vars_list))
+
+pytestmark = skip_tests(["GITHUB_TOKEN", "GITHUB_USER"])
```

Comment:
How does this `pytestmark` work? Is it documented somewhere? I can't find anything.